app/experimenter/experiments/tests/test_views.py

```diff
@@ -1110,6 +1110,36 @@ def test_view_saves_experiment(self):
         self.assertEqual(change.new_status, experiment.STATUS_DRAFT)
 
 
+class TestResultsUpdateView(TestCase):
+
+    def test_view_saves_experiment(self):
+        user_email = "user@example.com"
+        experiment = ExperimentFactory.create_with_status(
+            Experiment.STATUS_COMPLETE
+        )
+
+        data = {
+            "results_url": "https://example.com",
+            "results_lessons_learned": "Many lessons were learned.",
+        }
+
+        response = self.client.post(
+            reverse(
+                "experiments-results-update", kwargs={"slug": experiment.slug}
+            ),
+            data,
+            **{settings.OPENIDC_EMAIL_HEADER: user_email},
+        )
+        self.assertEqual(response.status_code, 302)
+
+        experiment = Experiment.objects.get()
+
+        self.assertEqual(experiment.results_url, "https://example.com")
+        self.assertEqual(
+            experiment.results_lessons_learned, "Many lessons were learned."
```

Comment:
Please reformat this to only use Yoda speak: 'Learned, many lessons were.' 

![source](https://user-images.githubusercontent.com/119884/62951059-44a77200-bdb7-11e9-990e-520019e96c09.gif)